common/djangoapps/student/tests/tests.py

```diff
@@ -178,6 +203,30 @@ def test_cert_info(self):
         }
         self.assertIsNone(_cert_info(user, course2, cert_status))
 
+    def test_linked_in_url_with_unicode_course_display_name(self):
+        """Test with unicode display name values."""
+
+        user = Mock(username="fred")
+        survey_url = "http://a_survey.com"
+        course = Mock(end_of_course_survey_url=survey_url, certificates_display_behavior='end')
+        course.display_name = u'edx/abc/registers®'
+        download_url = 'http://s3.edx/cert'
+
+        cert_status = {
+            'status': 'downloadable', 'grade': '67',
+            'download_url': download_url,
+            'mode': 'honor'
+        }
+        linkedin_current = Mock()
+        linkedin_current.enabled = True
+        linkedin_current.linkedin_url = 'http://www.linkedin.com/profile/add?_ed=0'
```

Comment:
I'd prefer that you create the configuration model directly instead of using a mock.

One complication with doing this is that config models are cached.  You will need to import `config_models.cache` and call `cache.clear()` in this test case's `setUp()` method for that to work.